src/wwwroot/controllers/PlanCtrl.js

```diff
@@ -174,6 +184,42 @@
       }      
     }
 
+    function isUpgradeablePlan(limit){
+      //Check constraints to validate that plan is upgradeable
+
+      //Set to true all errors to prevent access if service is not working
+      var errorsPlanUpgrade = {'isDKIMError': true, "isDeliveredError": true, "isDeliveredAndDKIMError": true};
+
+      if(limit==undefined){
+        var isConfigRequired = auth.getLimitsByAccount().then(isUpgradeablePlan).catch($log.error);
+      }
+      else{
+        //Exist activity in the current plan?
+        errorsPlanUpgrade.isDeliveredError = vm.totalDeliveriesCount > 0? false : true;
```

Comment:
Very minor, but

```js
errorsPlanUpgrade.isDeliveredError = vm.totalDeliveriesCount > 0? false : true;
``` 

Is the same than:

```js
errorsPlanUpgrade.isDeliveredError = vm.totalDeliveriesCount <= 0;
```

or
 
```
errorsPlanUpgrade.isDeliveredError = !(vm.totalDeliveriesCount > 0);
```

Reply:
Thanks I will implement it.